code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -376,7 +376,10 @@
 
 /datum/action/innate/ai/destroy_rcds/Activate()
 	for(var/obj/item/rcd/RCD in GLOB.rcd_list)
-		if(!istype(RCD, /obj/item/rcd/borg)) //Ensures that cyborg RCDs are spared.
+		var/turf/RCD_turf = get_turf(RCD)
+		if(istype(RCD, /obj/item/rcd/borg)) //Ensures that cyborg RCDs are spared.
+			continue
+		if(RCD_turf.z != 1)
```

Comment:
Reachable levels include: The station, lavaland, space levels, but no other levels
```suggestion
		if(is_level_reachable(RCD_turf.z))
```

Reply:
Done